### PR TITLE
Add support for pinning multiple domains with a single OkHttpClient

### PR DESCRIPTION
**Pull request by SteveShenouda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,20 +116,25 @@ protected void onCreate(Bundle savedInstanceState) {
   connection.setSSLSocketFactory(TrustKit.getInstance().getSSLSocketFactory(serverHostname));
 
   // OkHttp 2.x
+  RootTrustManager trustManager = new RootTrustManager();
```
Comment (nabla-c0d3) [inline README.md]: OkHttpHelper
* getSSLSocketFactory()
* getPinningInterceptor()

File: trustkit/src/main/java/com/datatheorem/android/trustkit/pinning/RootTrustManager.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    public void checkClientTrusted(X509Certificate[] chain, String authType) throws CertificateException {
+        TrustKit.getInstance().getTrustManager(mServerHostname.get()).checkClientTrusted(chain, authType);
+    }
+
+    @Override
+    public void checkServerTrusted(X509Certificate[] chain, String authType) throws CertificateException {
+        String host = mServerHostname.get();
+        DomainPinningPolicy serverConfig =
+                TrustKit.getInstance().getConfiguration().getPolicyForHostname(host);
+        if (serverConfig == null) {
```
Comment (nabla-c0d3) [inline trustkit/src/main/java/com/datatheorem/android/trustkit/pinning/RootTrustManager.java]: What's the need for the if/else here? What about `TrustKit.getInstance().getTrustManager(host).checkServerTrusted(chain, authType)` in all cases?

Comment (SteveShenouda) [reply trustkit/src/main/java/com/datatheorem/android/trustkit/pinning/RootTrustManager.java]: Added a comment to address the need

File: README.md (hunk 1 above)
Comment (SteveShenouda) [reply README.md]: Great idea! Done!

**Comment by nabla-c0d3** [comment]
Thanks!

**Pull request merged**